app/views/pages/support.html.erb

```diff
@@ -26,6 +26,11 @@
         For questions about a dataset, you can contact the publisher directly if they’ve provided contact details on the
         dataset page.
       </p>
+
+      <p>
+        If you have feedback about the site, fill in <%= link_to "the survey", "http://www.smartsurvey.co.uk/s/3SEXD/" %>.
```

Comment:
This seemed a little punchy to me - I now feel less inclined to give my feedback ;-) Perhaps we could add some fluff?

Reply:
The content is from Gurinder - I'll ask her to look at it

Reply:
In the meantime, I think it would be good to get these changes through so we can divert feedback to the survey. Content tweaks can come later